migration/sql-files/010-add-cluster-to-user.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users ADD COLUMN cluster TEXT;
+UPDATE users SET cluster = '{{.}}'  WHERE cluster is null or cluster = '';
```

Comment:
Let's add NOT NULL to the cluster field after updating the existing users.

Reply:
Thanks!
e13d57d71788b85bfa31808932b0554573218ffd